samples/crypto/aes_cbc/sample.yaml

```diff
@@ -46,7 +62,13 @@ tests:
       - nrf54l15dk/nrf54l05/cpuapp
   sample.aes_cbc.cracen.crypto_service:
     sysbuild: true
-    tags: introduction psa cracen sysbuild crypto ci_samples_crypto
+    tags:
+      - introduction
+      - psa
+      - cracen
+      - sysbuild
+      - crypto
+      - ci_samples_crypto
     platform_allow: >
       nrf54h20dk/nrf54h20/cpuapp
```

Comment:
crypto samples does not use YAML lists for 54h20 target

Reply:
This format is still supported. It is allowed to have strings for single items.